static/js/public/first-snap-flow.test.js

```diff
@@ -6,6 +6,9 @@ describe("initChooseName", () => {
   let nameInput;
   let submitButton;
 
+  const _window = window;
+  window.location = _window.location;
```

Comment:
This looks really hacky, is this fine?

Reply:
@solazio what the reason for this? It seems it's just reassigning the value from itself?

Reply:
the "good" way to do this would be `const { location } = window;`, but  babel-eslint module doesn't like that because further I I only use `window.location` and not `location` so it throws an error. To fix it I either hack it this way or I would probably need to find and install another module which under the hood will probably do the same thing.

Reply:
But I still don't understand the reason. if _windows = windows, then when you do `window.location = _window.location;` is exactly as it was before, right?

Reply:
fixed it now. thanks

Reply:
@solazio Thank you for the explanation in the call!